### ISSUE TITLE
Search Bar is not available on User dashboard

### ISSUE THREAD

**Issue by cam156** [samvera/hyrax#1245]
### Descriptive summary

In master When I login I get redirected to the dashboard.  When there I have no search capabilities, no help page or contact form.  Was this done intentionally?  I have to know that I need to go back to the home pages to get at the repository.

On the dashboard there is no search or menu bar:
![screen shot 2017-06-20 at 12 52 01 pm](https://user-images.githubusercontent.com/1599081/27345370-6f44052e-55b7-11e7-8d8b-9f2d6a6980c0.png)

On the home page a search and menu bar exists:
![screen shot 2017-06-20 at 12 52 10 pm](https://user-images.githubusercontent.com/1599081/27345355-64b28770-55b7-11e7-98ca-6f972da39315.png)

### Rationale

As a new user to hyrax I want to have a menu bar so I can quickly search the system, or ask for help if I am lost.

### Expected behavior

I would expect the menu section to be on most pages (if not all pages)

### Actual behavior

The menu bar disappears on the dashboard.

### Steps to reproduce the behavior

1. Go to the hyrax home page
1. Then login
1. Then the menu bar disappears.

**Comment by ggeisler** [comment]
When you are in the Admin section of the application, you can click on the application name (e.g., Hyrax) in the left-side of the topbar to return to the public side of the application.

If the community doesn't think that is sufficient to provide access to the menu and searchbox, one option would be to add it on the admin side just below the topbar:

![hyrax-mockups_graffle__admin__with_menu_bar___search_box](https://user-images.githubusercontent.com/101482/27346812-35c2da00-55a3-11e7-9b03-4361e9655bab.png)

On the one hand, I think this adds a bit of dissonance: you can do a search from the admin pages but the results are going to take you out of the admin context and put you in the public pages.

On the other hand, I think the single (likely not immediately obvious) link you need to follow to return to the public side if you want to search or access the main menu might confuse some (especially new) users.

**Comment by cam156** [comment]
@ggeisler I think there may be a disconnect.  I am not logged in as an administrator.  I do see Administration in my bread crumbs even if I am not an administrator, which may be related to Administration of the repository, not an Administration status.  

The disconnect is likely based on my point of view as a self deposit user of Hyrax, vs your point of view where only system administrators can login and deposit.

**Comment by ggeisler** [comment]
@cam156 I'm not sure if my conception of a self deposit user is incorrect, but just to make sure we are on the same page:

When I referred to the Admin section, I believe this is a section that would be seen by a self-deposit user, not just repository administrators. With the changes the Hyku team has made over the past year or so, we basically now have two distinct views of the application: the public side (pages that don't have the dark gray sidebar) and the admin side (pages that do have the dark gray sidebar). 

A site visitor with no privileges at all (not logged-in) would only ever see the public pages. Any logged-in user (an ordinary institution user -- say, a researcher -- or a repository admin with comprehensive privileges) will see both the public pages and some version of the admin pages. The screenshot in my previous comment would be a repository admin user. But below is what a non-admin, registered user would see when they want to manage their works or collections, etc. Note that this user has much fewer options showing in the sidebar, but still uses the admin side to do things like view and edit their works, collections, etc.

![hyrax-mockups_graffle__admin__with_menu_bar___search_box](https://user-images.githubusercontent.com/101482/27351482-5fb57020-55b2-11e7-874e-bc4e404d5908.png)

So the site menu + search box still might be something we should consider (it's in the above mockup but darker than the previous version)?

**Comment by mjgiarlo** [comment]
I wonder if it would help to refer to the collapsible-sidebar UI as "the dashboard" instead of as "admin pages." That might more clearly convey that this is the UI that is for interacting with Hyrax beyond end-user search and display. Integrating search capabilities into the dashboard, and making sure that search results originating from the dashboard search render the dashboard layout, seems like a reasonable feature request given this works in Sufia already. I find the `HOME ABOUT HELP CONTACT` bit jarring, personally, but if we already have a bar up there for search, I suppose we ought to use it to hold those sitewide links in a consistent spot.

**Comment by mjgiarlo** [comment]
I'd like more @samvera/hyrax-ui-ux-advisors to chime in on the above, too.

**Comment by newmanld** [comment]
I agree with @cam156 that it is confusing to have no search bar on the Dashboard.  

I can confirm that at Cincinnati we added the search bar above the "My Dashboard line" in our current implementation.  In our case we had some other menu items to promote as well on that line.  Screenshot below:
![image](https://user-images.githubusercontent.com/3521707/27358322-12a95260-55e4-11e7-9171-43992b2fd83a.png)

**Comment by cpd3149** [comment]
here's northwestern's registered user dashboard (search + help pages): 
![image](https://user-images.githubusercontent.com/24395592/27358448-4ff31122-55dc-11e7-8d6a-7ad410f6f1f5.png)

here's our admin page (search, no help pages):
![image](https://user-images.githubusercontent.com/24395592/27358476-73feb2ba-55dc-11e7-8f8d-822255d5164f.png)

we're on hyrax 1 in production.

**Comment by rjkati** [comment]
In Hyrax 3.0.2 the search bar and menu bar do not appear on the user dashboard.
